scripts/main.js

```diff
@@ -1 +1,7 @@
-console.log("Test");
+var birthdayInput = document.querySelector(".selection-intro #birthDate");
```

Comment:
We should use `var` as least as possible and stick with using `let` and `const` as `var` has a large scope. `const` would be the most appropriate here since the variable `birthdayInput` will not be modified.